wakuv2/waku.go

```diff
@@ -505,7 +506,7 @@ func (w *Waku) telemetryBandwidthStats(telemetryServerURL string) {
 				w.bandwidthCounter.Reset()
 			}
 
-			storeStats := w.bandwidthCounter.GetBandwidthForProtocol(store.StoreID_v20beta4)
+			storeStats := w.bandwidthCounter.GetBandwidthForProtocol(store.StoreQueryID_v300)
```

Comment:
Interesting, i was not aware bandwidth was already measured and pushed to telemetry. Nice.